modules/virt-removing-interface-from-nodes.adoc

```diff
@@ -14,9 +14,7 @@ If you remove a bridge or bonding interface, any node NICs in the cluster that w
 
 [NOTE]
 ====
-Deleting the node network policy that added an interface does not change the configuration of the policy on the node.
-Although a `NodeNetworkConfigurationPolicy` is an object in the cluster, it only represents the requested configuration. +
-Similarly, removing an interface does not delete the policy.
+Deleting the node network policy that added an interface does not change the configuration of the policy on the node. Although a `NodeNetworkConfigurationPolicy` is an object in the cluster, the object only represents the requested configuration. Similarly, removing an interface does not delete the policy.
```

Comment:
>Similarly, removing an interface does not delete the policy.

You are opening a can of worms with this sentence :)

If we want to pursue this path, we should add something like -- "[...] removing an interface does not delete the policy. In this case however, K-NMstate will recreate the deleted interface once its Pod or Node is restarted."